src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -429,106 +419,96 @@ limitations under the License.
+    <attribute code="0x004D" side="server" type="int8u" define="NUMBER_OF_SCHEDULE_TRANSITION_PER_DAY" isNullable="true" optional="true">NumberOfScheduleTransitionPerDay</attribute>
+    <attribute code="0x004E" side="server" type="octet_string" define="ACTIVE_PRESET_HANDLE" isNullable="true" length="16" optional="true">ActivePresetHandle</attribute>
+    <attribute code="0x004F" side="server" type="octet_string" define="ACTIVE_SCHEDULE_HANDLE" isNullable="true" length="16" optional="true">ActiveScheduleHandle</attribute>
+    <attribute code="0x0050" side="server" type="array" entryType="PresetStruct" define="PRESETS" writable="true" optional="true" mustUseAtomicWrite="true">
       <description>Presets</description>
       <access op="write" privilege="manage"/>
     </attribute>
-    <attribute code="0x0051" side="server" type="ARRAY" entryType="ScheduleStruct" define="SCHEDULES" writable="true" optional="true">
+    <attribute code="0x0051" side="server" type="array" entryType="ScheduleStruct" define="SCHEDULES" writable="true" optional="true" mustUseAtomicWrite="true">
       <description>Schedules</description>
       <access op="write" privilege="manage"/>
     </attribute>
-    <attribute code="0x0052" side="server" type="boolean" define="PRESETS_SCHEDULES_EDITABLE" writable="false" optional="true">PresetsSchedulesEditable</attribute>
-    <attribute code="0x0053" side="server" type="TemperatureSetpointHoldPolicyBitmap" define="TEMPERATURE_SETPOINT_HOLD_POLICY" default="0" writable="false" optional="true" min="0x0" max="0x3">TemperatureSetpointHoldPolicy</attribute>
-    <attribute code="0x0054" side="server" type="epoch_s" define="SETPOINT_HOLD_EXPIRY_TIMESTAMP" isNullable="true" writable="false" optional="true">SetpointHoldExpiryTimestamp</attribute>
-
+    <attribute code="0x0052" side="server" type="epoch_s" define="PRESETS_SCHEDULES_EDITABLE" optional="true" isNullable="true">SetpointHoldExpiryTimestamp</attribute>
     <!-- Client Commands -->
-
     <command source="client" code="0x00" name="SetpointRaiseLower" optional="false">
-      <description>
-        Command description for SetpointRaiseLower
-      </description>
-      <arg name="Mode" type="SetpointRaiseLowerModeEnum" min="0x0" max="0x2"/>
-      <arg name="Amount" type="int8s"/>
+      <description>Upon receipt, the attributes for the indicated setpoint(s) SHALL have the amount specified in the Amount field added to them.</description>
+      <arg id="0" name="Mode" type="SetpointRaiseLowerModeEnum" min="0x0" max="0x2"/>
+      <arg id="1" name="Amount" type="int8s"/>
     </command>
+
     <command source="client" code="0x01" name="SetWeeklySchedule" optional="true">
-      <description>
-        Command description for SetWeeklySchedule
-      </description>
+      <description>Upon receipt, the weekly schedule for updating setpoints SHALL be stored in the</description>
```

Comment:
Note: this is a really weird description that gets cut off.  Similar for SetActiveScheduleRequest and SetActivePresetRequest.  Needs some Alchemy fixes, perhaps? 

Followup PR to address these is fine.

Reply:
Filed https://github.com/project-chip/alchemy/issues/4